### PR TITLE
EPMRPP-85843 || Add API Key Link to documentation on Profile page

### PR DESCRIPTION
**Pull request by BlazarQSO**

### CODE REVIEW

File: app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.scss [hunk 1]
```diff
@@ -24,6 +24,14 @@
   font-size: 12px;
   color: $COLOR--gray-60;
   margin-bottom: 24px;
```
Comment (Bam6ycha) [inline app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.scss:26]: Please, add line-height property here

**Review by Bam6ycha — CHANGES_REQUESTED** [review]

File: app/localization/translated/uk.json [hunk 2]
```diff
@@ -159,12 +159,14 @@
   "ApiKeyGeneratedModal.header": "Ключ API створено",
   "ApiKeyGeneratedModal.loaderText": "ГЕНЕРУЮЧИЙ",
   "ApiKeyGeneratedModal.successNotification": "Ключ API успішно скопійовано",
-  "ApiKeys.ApiKeysBlock.description": "Щоб забезпечити безпеку власного пароля домену, ви можете використовувати ключ користувача — щоб підтвердити свій обліковий запис і мати можливість надсилати результати за допомогою агента.",
+  "ApiKeys.ApiKeysBlock.description": "Щоб забезпечити безпеку власного пароля домену, ви можете використовувати ключ користувача — щоб підтвердити свій обліковий запис і мати можливість увійти за допомогою агента. Більш детальну інформацію про ключі API ви можете прочитати в",
+  "ApiKeys.ApiKeysBlock.descriptionLink": "Документації",
```
[the remark this answers is not in the document]
Comment (BlazarQSO) [reply app/localization/translated/uk.json]: Done.

File: app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.scss [hunk 3]
```diff
@@ -24,6 +24,14 @@
...
+  max-width: 780px;
+}
+
+.description-link {
+  font-family: $FONT-REGULAR;
+  font-size: 12px;
+  color: $COLOR--topaz;
+  text-decoration: none;
```
[the remark this answers is not in the document]
Comment (BlazarQSO) [reply app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.scss]: Done.

File: app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.scss [hunk 4]
```diff
@@ -24,6 +24,14 @@
   font-size: 12px;
   color: $COLOR--gray-60;
   margin-bottom: 24px;
+  max-width: 780px;
```
[the remark this answers is not in the document]
Comment (BlazarQSO) [reply app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.scss]: Done.

**Review by Bam6ycha — APPROVED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**